cpp/src/interop/from_arrow_host.cu

```diff
@@ -252,6 +252,10 @@ std::unique_ptr<column> dispatch_copy_from_arrow_host::operator()<cudf::list_vie
   ArrowSchemaView view;
   NANOARROW_THROW_NOT_OK(ArrowSchemaViewInit(&view, offset_schema.get(), nullptr));
 
+  CUDF_EXPECTS(input->length + input->offset + 1 <=
+                 static_cast<std::int64_t>(std::numeric_limits<cudf::size_type>::max()),
```

Comment:
I think this still overflow.
```suggestion
CUDF_EXPECTS(static_cast<std::int64_t>(input->length) + static_cast<std::int64_t>(input->offset) + 1L <=
                 static_cast<std::int64_t>(std::numeric_limits<cudf::size_type>::max()),
```

Reply:
I think `input->length` and `input->offset` are already int64_t

Reply:
Then it is all good.

Reply:
[Yes, they are indeed](https://arrow.apache.org/docs/format/CDataInterface.html#the-arrowarray-structure).